fighthealthinsurance/rest_views.py

```
@@ -1527,11 +1528,49 @@ def perform_create(self, request: Request, serializer) -> Response:
+                clicked_for_paid=False,
+            )
+            notify_interested_professional(
+                interested_pro,
+                source="a Fight Paperwork demo request",
+                subject=f"New demo request lead #{interested_pro.id}",
+            )
+            # Send the thank-you right away; dosend() sets
+            # thankyou_email_sent=True on success so the batched
+            # ThankyouEmailSender won't re-send, and it swallows mail errors.
+            ThankyouEmailSender().dosend(interested_pro=interested_pro)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)